ops/model.py

```diff
@@ -405,7 +574,21 @@ def __repr__(self):
 
 
 class RelationData(Mapping):
-    def __init__(self, relation, our_unit, backend):
+    """Represents the various data buckets of a given relation.
+
+    Each unit and application involved in a relation has their own data bucket.
+    Eg: {entity: RelationDataContent}
```

Comment:
```suggestion
    E.g.: ``{entity: RelationDataContent}``
```

Reply:
or maybe even
```suggestion
    E.g.::
    
        {entity: RelationDataContent}
    
```

Reply:
Shouldn't Sphinx complain if there is something wrong?

Reply:
I realize now this isn't that it was incorrect syntax, as much as 'I'd like to see it as preformatted code'.
I did go that route for this one, but not for all the others. I'm not 100% satisfied with how it is reported, but this map is a lot more complex than the others, so they get simple type references. But using English (`map of foo to :class:Bars`)